interactors/nonceHandlerV3/addressNonceHandler.go

```diff
@@ -55,18 +56,19 @@ func NewAddressNonceHandlerV3(proxy interactors.Proxy, address sdkCore.AddressHa
 // ApplyNonceAndGasPrice will apply the computed nonce to the given FrontendTransaction
 func (anh *addressNonceHandler) ApplyNonceAndGasPrice(ctx context.Context, txs ...*transaction.FrontendTransaction) error {
```

Comment:
no, you didn't miss anything. I didn't think of applying the nonce and gas price asyncronously, but I think it'd make some sense to do it. Though, this will need some refactoring and rethinking.